viskores/exec/CellLocatorBoundingIntervalHierarchy.h

```diff
@@ -191,10 +196,16 @@ class VISKORES_ALWAYS_EXPORT CellLocatorBoundingIntervalHierarchy
 
   /// @copydoc viskores::exec::CellLocatorUniformGrid::FindAllCells
   template <typename CellIdsType, typename ParametricCoordsVecType>
-  VISKORES_EXEC viskores::ErrorCode FindAllCells(const viskores::Vec3f& point,
-                                                 CellIdsType& cellIdsVec,
-                                                 ParametricCoordsVecType& pCoordsVec) const
+  VISKORES_EXEC viskores::ErrorCode FindAllCells(const viskores::Vec3f& viskoresNotUsed(point),
+                                                 CellIdsType& viskoresNotUsed(cellIdsVec),
+                                                 ParametricCoordsVecType& viskoresNotUsed(pCoordsVec)) const
   {
+    //There is a memory access error on some GPU devices.
+    // Disabling for now.
+    printf("Warning: CellLocatorBoundingIntervalHierarchy::FindAllCells() is disabled.\n");
```

Comment:
Although I suggested using `printf` for debugging in a worklet, merging code with it in is a bad idea. Both Cuda and HIP will compile, but it can drastically increase the time and memory the compiles take. And some compilers, like SYCL, may not compile at all.

So, while temporarily using it to see state is OK, keeping it in the code is bad.